### ISSUE TITLE
[REQ] Create a new note with name of my choice

### ISSUE THREAD

**Issue by peavine** [glushchenko/fsnotes#520]
I frequently create new notes and would like a simpler way to do this with note names of my choice. I proposed one alternative in #511. Another approach is:

1) The user selects "File > New" from the top menu or the command-N keyboard shortcut. This would place the focus in what I guess is called the title bar.

2) The note name shown in the title bar would be "Untitled Note". This note name would be highlighted so that the user could type a note name of their choosing. The Finder works this way when creating a new folder.

3) The user selects the Enter key to create a note with the name "Untitled Note". Alternatively, the user types a note name of their choosing and then selects the Enter key to create a note with that name. 

This request assumes that the preference option to "Use first line as title" is not enabled.

Thanks.

**Comment by gingerbeardman** [comment]
My preference would be to make the name of the note automatic, but with manual override. 

I don't want to waste time naming add renaming notes, so I'd prefer it if the note automatically took on the name of the first heading. 

If the note is manually renamed to be different to the first heading, then it's name would remain under manual control until it's either renamed to match or name is deleted competely so it returns to default automatic naming.  

Basically, make it one more thing for the computer to take care off, one less thing for me to be concerned with.

**Comment by peavine** [comment]
FWIW, there is a preference option to "Use first line as title". This option is set folder-by-folder and does not do exactly what @gingerbeardman wants, but it is there.

**Comment by gingerbeardman** [comment]
Good point. Though I've never been able to get that option to work.

**Comment by peavine** [comment]
The new options to 1) have FSNotes start with the cursor in the Search and Create bar and 2) the command-n keyboard shortcut make the creation of a new note very simple. So, this request is no longer desired. Thanks.
